<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ldmt_display</class>
 <widget class="QMainWindow" name="ldmt_display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Display</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(68,68,68);
}
QWidget#centralwidget{
background-color: rgb(68,68,68);
}
QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}

QRadioButton{
color:rgb(175,175,175);
}
QCheckBox{
color:rgb(175,175,175);
}

QStatusBar QLabel {
    border: 3px solid white;
}

QStatusBar {
    color: rgb(175,175,175);
}

QStatusBar::item {
    border: 1px solid red;
    border-radius: 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>70</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="2" column="4">
     <widget class="QPushButton" name="btn_count">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Display scene info.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">*.QPushButton{ 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(135, 135, 135, 225), stop:1 rgba(185,185, 185, 195));
border:1px solid rgb(30,30,30);
color: rgb( 25,25,25 );
}
*.QPushButton:hover{ 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(155, 155, 155, 225), stop:1 rgba(205,205, 205, 195));
border:1px solid rgb(30,30,30);
color: rgb( 25,25,25 );
text-decoration: underline!important;
}
*.QPushButton:pressed{ 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(175, 175, 175, 195), stop:1 rgba(115,115, 115, 225));
border:1px solid rgb(30,30,30);
color: rgb( 25,25,25 );
}</string>
      </property>
      <property name="text">
       <string>Count</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn_vertNormal">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Display vert normal.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}
</string>
      </property>
      <property name="text">
       <string>Vert Normal</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="btn_reset">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Reset all display.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">*.QPushButton{ 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(135, 135, 135, 225), stop:1 rgba(185,185, 185, 195));
border:1px solid rgb(30,30,30);
color: rgb( 25,25,25 );
}
*.QPushButton:hover{ 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(155, 155, 155, 225), stop:1 rgba(205,205, 205, 195));
border:1px solid rgb(30,30,30);
color: rgb( 25,25,25 );
text-decoration: underline!important;
}
*.QPushButton:pressed{ 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(175, 175, 175, 195), stop:1 rgba(115,115, 115, 225));
border:1px solid rgb(30,30,30);
color: rgb( 25,25,25 );
}</string>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn_texBorder">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Display texture border.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}
</string>
      </property>
      <property name="text">
       <string>Tex Border</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btn_faceNormal">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Display face normal.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}
</string>
      </property>
      <property name="text">
       <string>Face Normal</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="btn_uvId">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Display uv id.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}
</string>
      </property>
      <property name="text">
       <string>UV Id</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btn_edgeId">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Display edge id.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}
</string>
      </property>
      <property name="text">
       <string>Edge Id</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="btn_faceId">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Display face id.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}
</string>
      </property>
      <property name="text">
       <string>Face Id</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn_vertId">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Display vert id.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}
</string>
      </property>
      <property name="text">
       <string>Vert Id</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btn_triangle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Display triangle split.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}
</string>
      </property>
      <property name="text">
       <string>Triangle</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="btn_normalSize">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Edit normal size.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">*.QPushButton{ 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(135, 135, 135, 225), stop:1 rgba(185,185, 185, 195));
border:1px solid rgb(30,30,30);
color: rgb( 25,25,25 );
}
*.QPushButton:hover{ 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(155, 155, 155, 225), stop:1 rgba(205,205, 205, 195));
border:1px solid rgb(30,30,30);
color: rgb( 25,25,25 );
text-decoration: underline!important;
}
*.QPushButton:pressed{ 
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(175, 175, 175, 195), stop:1 rgba(115,115, 115, 225));
border:1px solid rgb(30,30,30);
color: rgb( 25,25,25 );
}</string>
      </property>
      <property name="text">
       <string>Normal Size</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="btn_tangent">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="statusTip">
       <string>Display tangent.</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
border:1px solid rgb(30,30,30);
border-radius:4px;
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(235, 158, 108, 195));
color:rgb(25,25,25);}
QPushButton:hover{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(189, 118, 57, 225), stop:1 rgba(255, 178, 128, 195));
color:rgb(25,25,25);}
QPushButton:pressed{ 
 border:1px solid rgb(30,30,30);
background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(235, 158, 108, 195), stop:1 rgba( 189, 118, 57, 225));
color:rgb(25,25,25);}
</string>
      </property>
      <property name="text">
       <string>Tangent</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
